zanata-overlay/distros/eap-6/standalone/configuration/standalone-zanata.xml

```diff
@@ -409,7 +409,7 @@
                 </security-domain>
                 <security-domain name="zanata.internal">
                     <authentication>
-                        <login-module code="org.jboss.seam.security.jaas.SeamLoginModule" flag="required"/>
+                        <login-module code="org.zanata.security.jaas.InternalLoginModule" flag="required"/>
```

Comment:
- [ ] Make sure you put this change in the release notes.  
- [ ] We will need to ensure this gets into Ansible too, at the right time.